.github/workflows/single-runner.yml

```diff
@@ -8,31 +8,39 @@ on:
         required: true
         default: dcp_mappluto
       latest:
+        type: boolean
         description: "Tag this version as latest (optional)"
         required: false
         default: "true"
       version:
         description: "The version of the dataset (i.e. 22v2, 21C) if needed (optional)"
         required: false
 
-
-
 jobs:
-  dataloading:
+  setup:
+    uses: ./.github/workflows/setup.yml
```

Comment:
all uses of a file from the repo in a workflow step _might_ first require `- uses: actions/checkout@v3`

but if it works without it then nvm!

Reply:
This was reverted